### PR TITLE
Revised learning objectives

### PR DESCRIPTION
**Pull request by kariljordan**
@tracykteal @cbahlai 
I'm submitting PRs to revise the learning objectives such that they align with Data Carpentry's assessment efforts.

### CODE REVIEW

File: 04-quality-control.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ Authors:**Christie Bahlai**, **Aleksandra Pawlik**<br>
 Contributors: **Jennifer Bryan**, **Alexander Duryee**, **Jeffrey Hollister**, **Daisie Huang**, **Owen Jones**,
 **Ben Marwick**, and **Ethan White**
 
+## Learning Objective 
+
+* Apply quality control techniques to ensure data in spreadsheets is free from errors.
+
```
Comment (tracykteal) [inline 04-quality-control.md]: We can't ensure things are free from error, so maybe?

Apply quality control techniques to identify errors in spreadsheets and limit incorrect data entry

Comment (kariljordan) [inline 04-quality-control.md]: @tracykteal Great call. Should I change that objective and submit a new PR?

**Comment by kariljordan** [comment]
Hi @tracykteal and @cbahlai, I edited the language for this learning objective. Can you merge the changes?

**Comment by cbahlai** [comment]
Looks good to me! I apologize for the delay, WAY backlogged on github notifications!

**Pull request merged**